CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.50.3
+* Enhancement - Eliminate benign docker stats "context canceled" warning messages from logs [#2813](https://github.com/aws/amazon-ecs-agent/pull/2813)
+* Bug - Fix bug where pause container was not always cleaned up [#2824](https://github.com/aws/amazon-ecs-agent/pull/2824)
+* Enhancement - Change amazon dockerhub images to public.ecr.aws, replace gcr ref [#2826](https://github.com/aws/amazon-ecs-agent/pull/2826)
```

Comment:
Im not sure if we want to be listing this on the changelog. This change was only for changing the image references of some of our integration tests so shouldn't affect any customers. Just worried if it's in the changelog some customers might misinterpret to mean that we are modifying or replacing their dockerhub images.